src/main/java/org/folio/rest/impl/VendorHelper.java

```diff
@@ -59,15 +62,15 @@ public VendorHelper(Map<String, String> okapiHeaders, Context ctx, String lang)
         .thenApply(vendor -> {
           VendorStatus status = VendorStatus.valueOf(vendor.getVendorStatus().toUpperCase());
           if(status != VendorStatus.ACTIVE) {
-            errors.add(createErrorWithId(ORDER_VENDOR_IS_INACTIVE, id));
+            errors.add(createErrorWithId(ORDER_VENDOR_IS_INACTIVE, id, poLines));
```

Comment:
vendor is at the PO level, I'm not sure it makes sense to add an error for every single POL

Reply:
@craigmcnally, done. Thanks!